n3fit/src/n3fit/io/writer.py

```diff
@@ -519,44 +514,44 @@ def evln2lha(evln):
         - 2 * evln[8]
     ) / 120
 
-    lha[10] = (
-        10 * evln[1]
-        - 15 * evln[11]
-        + 3 * evln[12]
-        + 2 * evln[13]
-        + 10 * evln[3]
-        - 15 * evln[6]
-        + 3 * evln[7]
-        + 2 * evln[8]
-    ) / 120
-
-    lha[2] = (
-        10 * evln[1]
-        - 15 * evln[11]
-        + 3 * evln[12]
-        + 2 * evln[13]
-        - 10 * evln[3]
-        + 15 * evln[6]
-        - 3 * evln[7]
-        - 2 * evln[8]
-    ) / 120
-
-    lha[11] = (5 * evln[1] - 6 * evln[12] + evln[13] + 5 * evln[3] - 6 * evln[7] + evln[8]) / 60
-
-    lha[1] = (5 * evln[1] - 6 * evln[12] + evln[13] - 5 * evln[3] + 6 * evln[7] - evln[8]) / 60
-
-    lha[12] = (evln[1] - evln[13] + evln[3] - evln[8]) / 12
+    # charm
```

Comment:
```suggestion

    # if a heavy quark is not active at Q0 (the scale at which the output of the fit is stored),
    # keep the PDF values at 0.0 to prevent small negative values due to numerical instabilities

    # charm
```